pylmod/tests/test_membership.py

```diff
@@ -2,23 +2,161 @@
 """
 Test the membership class
 """
+import httpretty
+import json
+from mock import patch
 from pylmod import Membership
+from pylmod.exceptions import (
+    PyLmodUnexpectedData,
+)
+
 from pylmod.tests.common import BaseTest
 
 
-class TestGradebook(BaseTest):
+class TestMembership(BaseTest):
     """Validate defined methods and constructors in Membership class"""
 
+    # Unit tests generally should do protected-accesses
+    # pylint: disable=protected-access
+    COURSE_ID = 12345
+
+    STAFF_BODY = {
+        u'response':
+        {
+            u'docs':
+            [
+                {
+                    u'displayName': u'Huey Duck',
+                    u'role': u'TA',
+                    u'sortableDisplayName': u'Duck, Huey'
+                },
+                {
+                    u'displayName': u'Louie Duck',
+                    u'role': u'CourseAdmin',
+                    u'sortableDisplayName': u'Duck, Louie'
+                },
+                {
+                    u'displayName': u'Benjamin Franklin',
+                    u'role': u'CourseAdmin',
+                    u'sortableDisplayName': u'Franklin, Benjamin'
+                },
+                {
+                    u'displayName': u'George Washington',
+                    u'role': u'Instructor',
+                    u'sortableDisplayName': u'Washington, George'
+                },
+            ],
+        },
+    }
+
+    COURSE_DATA = {
+        u'response':
+        {
+            u'docs':
+            [
+                {
+                    u'googleEnabled': True,
+                    u'groupTemplate': u'project',
+                    u'groupingScheme': None,
+                    u'id': 12345,
+                    u'longName': u'pylmod test Site',
+                    u'name': u'pylmod test Site',
+                    u'properties':
+                    {
+                        u'allowsStudentsToSwitchSection': u'1',
+                        u'lastDateToSwitchSection': u'1480723199000'
+                    },
+                    u'termCode': u'2033FA',
+                    u'uuid': u'/project/testingstuff'
+                }
+            ],
+        },
+    }
+
+    def _register_get_course_id(self, body):
+        """Handle API call to get course id"""
+        httpretty.register_uri(
+            httpretty.GET,
+            '{0}courseguide/course?uuid={1}'.format(
+                self.MEMBERSHIP_REGISTER_BASE,
+                self.CUUID,
+            ),
+            body=json.dumps(body)
+        )
+
     def test_constructor(self):
         """Verify constructor does as expected i.e. append URLBASE with
         ``service/membership/``
         """
         # Strip off base URL to make sure it comes back
         urlbase = self.URLBASE[:-1]
-        test_membership = Membership(self.CERT, urlbase)
+        test_base = Membership(self.CERT, cuuid=None, urlbase=urlbase)
```

Comment:
The variable name since it is an instance of the Membership class, not the Base class.  Also, just curious, is this new and I didn't notice, or did you recently change the variable name?